fimsauthz-api/fims-authz.yaml

```diff
@@ -291,7 +320,23 @@ components:
             For this server, this is satisfied by the use of a UUID.
         scope:
           description: See securityDefinitions section in this spec.
+          type: array
+          items:
+            type: string
+            example: "example.scope.name.read"
+          maxItems: 1
+          minItems: 1
+        authorities:
+          description: An array of all authorities that the client is associated with.
+          type: array
+          items:
+            type: string
+            example: "USS_BASIC"
```

Comment:
Recommend using ASTM scope.

Also in https://github.com/nasa/utm-apis/blob/master/fimsauthz-api/nup-roles.yaml  , because the old UTM scopes are used as a temp/bridge solution they do not need to be in the public Roles file.  Request deleting them.